services/plans/plan_templates/schemas.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/leanspace/terraform-provider-leanspace/helper/general_objects"
 )
 
-var nameRegex = regexp.MustCompile(`^[a-zA-Z0-9_-]*$`)
+var nameRegex = regexp.MustCompile(`^[ a-zA-Z0-9_-]*$`)
```

Comment:
example on https://regexr.com/ with `Standard Imaging Sequence`. Spaces are not accepted with the current regex